src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java

```diff
@@ -89,7 +89,7 @@ public void onBuildTriggered(AbstractProject<?, ?> project, String commitSha, bo
         String triggeredStatus = getDescriptor().getTriggeredStatusDefault(this);
 
         // check if we even need to update
-        if (triggeredStatus.equals("--none--")) {
+        if (triggeredStatus != null && triggeredStatus.equals("--none--")) {
```

Comment:
No, if triggeredStatus is null or empty then line 105 will make it ```Build triggered.``` with extra text as decided by ```isMergeable```